src/Tests/dotnet-watch.Tests/StaticFileHandlerTest.cs

```diff
@@ -36,7 +36,7 @@ public async ValueTask TryHandleFileAction_WritesUpdateCssMessage()
         }
 
         [Fact]
-        public async ValueTask TryHandleFileAction_CausesBrowserRefreshForNonCssFile()
+        public async Task TryHandleFileAction_CausesBrowserRefreshForNonCssFile()
```

Comment:
These two tests never ran before because they returned ValueTaks which xunit doesn't support. They are failing because L16 and L42 pass a sealed type to Moq, which isn't supported (and the passed in ctor arguments don't seem to match either). Any opinions on what to do with those two tests? Delete them, comment them out or try to fix them?

Reply:
If they never ran before I would say delete them and log an issue pointing to the commit of the deletion.

Reply:
https://github.com/dotnet/sdk/issues/35971